setup.py

```diff
@@ -37,7 +37,7 @@
     # Used for oscar.test.newfactories
```

Comment:
@maikhoepfel I have a question here do you need a pin `<5.2` since `5.1` brings the django32 support ? This constraint brings the package with `django-phonenumber-field==5.0.0`.
https://github.com/stefanfoulis/django-phonenumber-field/blob/main/CHANGELOG.rst#510-2021-04-02